src/test/java/hudson/plugins/sshslaves/verifiers/TrustHostKeyActionTest.java

```diff
@@ -144,5 +140,113 @@ public boolean authenticate(String username, String password, ServerSession sess
         
         
     }
-    
+
+    private Object newSshServer() throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        Object server = null;
+        Class serverClass;
+        try {
+            serverClass = Class.forName("org.apache.sshd.SshServer");
+        } catch (ClassNotFoundException e) {
+            serverClass = Class.forName("org.apache.sshd.server.SshServer");
```

Comment:
You need to edit `Jenkinsfile` to say, e.g.

```groovy
buildPlugin(jenkinsVersions: [null, '2.89.1'])
```

so that CI actually tests both branches.

Reply:
OK! But as version '2.73.1' shares the same sshd-core library than '2.89.1', I think it will be better to use it